src/test/resources/network/brightspots/rcv/test_data/hart_travis_county_officers/hart_travis_county_officers_config.json

```diff
@@ -0,0 +1,68 @@
+{
+  "tabulatorVersion" : "1.1.0",
+  "outputSettings" : {
+    "contestName" : "TRAVIS COUNTY OFFICERS",
+    "outputDirectory" : "output",
+    "contestDate" : "",
+    "contestJurisdiction" : "",
+    "contestOffice" : "",
+    "tabulateByPrecinct" : false,
+    "generateCdfJson" : false
+  },
+  "cvrFileSources" : [ {
+    "filePath" : "CVRArchive_18_9_0_6298/",
```

Comment:
This test fails for me with:
>2020-07-08 12:20:29 PDT INFO: Reading Hart cast vote record file: 04917dca-3f9e-440e-a41a-98dc5e57376e.xml...
> 2020-07-08 12:20:29 PDT SEVERE: Error parsing cast vote record:
com.fasterxml.jackson.core.JsonParseException: Unexpected first character (char code 0xEF), not valid in xml document: could be mangled UTF-8 BOM marker. Make sure that the Reader uses correct encoding or pass an InputStream instead
> 2020-07-08 12:20:29 PDT SEVERE: Error opening cast vote record file: C:\Users\UserName\IdeaProjects\rcv\src\test\resources\network\brightspots\rcv\test_data\hart_travis_county_officers\CVRArchive_18_9_0_6298
> 2020-07-08 12:20:29 PDT INFO: Check file path and permissions and make sure they are correct!

Reply:
Can you post the preceding log output?  I'm wondering if it only happens for this file or all of them.

Reply:
Do these tests not fail for you? Here's the whole thing:
```
2020-07-09 22:21:11 PDT INFO: Starting tabulation session...
2020-07-09 22:21:11 PDT INFO: Successfully loaded contest config: C:\Users\UserName\IdeaProjects\rcv\src\test\resources\network\brightspots\rcv\test_data\hart_travis_county_officers\hart_travis_county_officers_config.json
2020-07-09 22:21:11 PDT INFO: Validating contest config...
2020-07-09 22:21:11 PDT INFO: Contest config validation successful.
2020-07-09 22:21:11 PDT INFO: Tabulation logging to: C:\Users\UserName\IdeaProjects\rcv\src\test\resources\network\brightspots\rcv\test_data\hart_travis_county_officers\output\2020-07-09_22-21-11_audit_*.log
2020-07-09 22:21:11 PDT INFO: Computer name: SMILODON
2020-07-09 22:21:11 PDT INFO: User name: UserName
2020-07-09 22:21:11 PDT INFO: Begin config file contents:
2020-07-09 22:21:11 PDT INFO: {
2020-07-09 22:21:11 PDT INFO:   "tabulatorVersion" : "1.1.0",
2020-07-09 22:21:11 PDT INFO:   "outputSettings" : {
2020-07-09 22:21:11 PDT INFO:     "contestName" : "TRAVIS COUNTY OFFICERS",
2020-07-09 22:21:11 PDT INFO:     "outputDirectory" : "output",
2020-07-09 22:21:11 PDT INFO:     "contestDate" : "",
2020-07-09 22:21:11 PDT INFO:     "contestJurisdiction" : "",
2020-07-09 22:21:11 PDT INFO:     "contestOffice" : "",
2020-07-09 22:21:11 PDT INFO:     "tabulateByPrecinct" : false,
2020-07-09 22:21:11 PDT INFO:     "generateCdfJson" : false
2020-07-09 22:21:11 PDT INFO:   },
2020-07-09 22:21:11 PDT INFO:   "cvrFileSources" : [ {
2020-07-09 22:21:11 PDT INFO:     "filePath" : "CVRArchive_18_9_0_6298/",
2020-07-09 22:21:11 PDT INFO:     "firstVoteColumnIndex" : "1",
2020-07-09 22:21:11 PDT INFO:     "firstVoteRowIndex" : "1",
2020-07-09 22:21:11 PDT INFO:     "idColumnIndex" : "1",
2020-07-09 22:21:11 PDT INFO:     "precinctColumnIndex" : "1",
2020-07-09 22:21:11 PDT INFO:     "provider" : "HART"
2020-07-09 22:21:11 PDT INFO:   } ],
2020-07-09 22:21:11 PDT INFO:   "candidates" : [
2020-07-09 22:21:11 PDT INFO:     {
2020-07-09 22:21:11 PDT INFO:     "name" : "Michael Gottner",
2020-07-09 22:21:11 PDT INFO:     "code" : "bdc77efc-2f86-48d0-a0de-2efb02e1f4d9",
2020-07-09 22:21:11 PDT INFO:     "excluded" : false
2020-07-09 22:21:11 PDT INFO:   },
2020-07-09 22:21:11 PDT INFO:     {
2020-07-09 22:21:11 PDT INFO:       "name" : "Jim Sylvester",
2020-07-09 22:21:11 PDT INFO:       "code" : "bfb4eda8-35e8-4d13-b152-99b8e5d6b777",
2020-07-09 22:21:11 PDT INFO:       "excluded" : false
2020-07-09 22:21:11 PDT INFO:     },
2020-07-09 22:21:11 PDT INFO:     {
2020-07-09 22:21:11 PDT INFO:       "name" : "William Poole",
2020-07-09 22:21:11 PDT INFO:       "code" : "24d59598-4ded-428f-842c-08921c8ed039",
2020-07-09 22:21:11 PDT INFO:       "excluded" : false
2020-07-09 22:21:11 PDT INFO:     },
2020-07-09 22:21:11 PDT INFO:     {
2020-07-09 22:21:11 PDT INFO:       "name" : "Bruce Elfant",
2020-07-09 22:21:11 PDT INFO:       "code" : "fa9ebe45-dd38-423d-a2a8-0bc2de9967f9",
2020-07-09 22:21:11 PDT INFO:       "excluded" : false
2020-07-09 22:21:11 PDT INFO:     },
2020-07-09 22:21:11 PDT INFO:     {
2020-07-09 22:21:11 PDT INFO:       "name" : "Sheriff Hamilton",
2020-07-09 22:21:11 PDT INFO:       "code" : "39a32be4-1804-467c-bf3e-06b174f892d7",
2020-07-09 22:21:11 PDT INFO:       "excluded" : false
2020-07-09 22:21:11 PDT INFO:     }
2020-07-09 22:21:11 PDT INFO:   ],
2020-07-09 22:21:11 PDT INFO:   "rules" : {
2020-07-09 22:21:11 PDT INFO:     "tiebreakMode" : "random",
2020-07-09 22:21:11 PDT INFO:     "overvoteRule" : "exhaustImmediately",
2020-07-09 22:21:11 PDT INFO:     "winnerElectionMode" : "standard",
2020-07-09 22:21:11 PDT INFO:     "randomSeed" : "9",
2020-07-09 22:21:11 PDT INFO:     "numberOfWinners" : "1",
2020-07-09 22:21:11 PDT INFO:     "multiSeatBottomsUpPercentageThreshold" : "",
2020-07-09 22:21:11 PDT INFO:     "decimalPlacesForVoteArithmetic" : "4",
2020-07-09 22:21:11 PDT INFO:     "minimumVoteThreshold" : "0",
2020-07-09 22:21:11 PDT INFO:     "maxSkippedRanksAllowed" : "1",
2020-07-09 22:21:11 PDT INFO:     "maxRankingsAllowed" : "max",
2020-07-09 22:21:11 PDT INFO:     "nonIntegerWinningThreshold" : false,
2020-07-09 22:21:11 PDT INFO:     "hareQuota" : false,
2020-07-09 22:21:11 PDT INFO:     "batchElimination" : false,
2020-07-09 22:21:11 PDT INFO:     "exhaustOnDuplicateCandidate" : false,
2020-07-09 22:21:11 PDT INFO:     "treatBlankAsUndeclaredWriteIn" : false,
2020-07-09 22:21:11 PDT INFO:     "overvoteLabel" : "",
2020-07-09 22:21:11 PDT INFO:     "undervoteLabel" : "",
2020-07-09 22:21:11 PDT INFO:     "undeclaredWriteInLabel" : "",
2020-07-09 22:21:11 PDT INFO:     "rulesDescription" : ""
2020-07-09 22:21:11 PDT INFO:   }
2020-07-09 22:21:11 PDT INFO: }
2020-07-09 22:21:11 PDT INFO: End config file contents.
2020-07-09 22:21:11 PDT INFO: Tabulating 'TRAVIS COUNTY OFFICERS'...
2020-07-09 22:21:11 PDT INFO: Parsing cast vote records...
2020-07-09 22:21:11 PDT INFO: Reading Hart cast vote records from folder: C:\Users\UserName\IdeaProjects\rcv\src\test\resources\network\brightspots\rcv\test_data\hart_travis_county_officers\CVRArchive_18_9_0_6298...
2020-07-09 22:21:11 PDT INFO: Reading Hart cast vote record file: 04917dca-3f9e-440e-a41a-98dc5e57376e.xml...
2020-07-09 22:21:11 PDT SEVERE: Error parsing cast vote record:
com.fasterxml.jackson.core.JsonParseException: Unexpected first character (char code 0xEF), not valid in xml document: could be mangled UTF-8 BOM marker. Make sure that the Reader uses correct encoding or pass an InputStream instead
2020-07-09 22:21:11 PDT SEVERE: Error opening cast vote record file: C:\Users\UserName\IdeaProjects\rcv\src\test\resources\network\brightspots\rcv\test_data\hart_travis_county_officers\CVRArchive_18_9_0_6298
2020-07-09 22:21:11 PDT INFO: Check file path and permissions and make sure they are correct!
2020-07-09 22:21:11 PDT SEVERE: Parsing cast vote records failed!
2020-07-09 22:21:11 PDT SEVERE: Aborting tabulation due to cast vote record errors!
2020-07-09 22:21:11 PDT INFO: Tabulation session completed.
2020-07-09 22:21:11 PDT INFO: Successfully loaded contest config: C:\Users\UserName\IdeaProjects\rcv\src\test\resources\network\brightspots\rcv\test_data\hart_travis_county_officers\hart_travis_county_officers_config.json
2020-07-09 22:21:11 PDT SEVERE: File not found!
java.io.FileNotFoundException: C:\Users\UserName\IdeaProjects\rcv\src\test\resources\network\brightspots\rcv\test_data\hart_travis_county_officers\output\2020-07-09_22-21-11_summary.json (The system cannot find the file specified)
```

Reply:
Yeah, they are passing fine for me.  I'll see if there is some stray byte here somehow.

Reply:
@HEdingfield There is indeed a BOM byte order mark on these xml files, which is no problem running on mac, but does seem like a known thing: https://stackoverflow.com/questions/1835430/byte-order-mark-screws-up-file-reading-in-java 

I pushed some test code to HartCvrReader:readCastVoteRecord.  Can you give that a shot and see if Hart tests complete on Windows?  There is another option which is  apache io commons has a file stream BOM reader, which we can pull in if needed: https://commons.apache.org/proper/commons-io/javadocs/api-2.2/org/apache/commons/io/input/BOMInputStream.html

Reply:
The initial fix didn't work, but the other one you left in there with the file stream did! Pushed commit where all my tests are passing now. If they pass on your end too, I think we're good to go!